### PR TITLE
Scenario Editor, Warg & minor fixes

### PR DESCRIPTION
**Pull request by Byak0**
big one aled :s

**New Scenario Editor**
Press O in the Modding Kit to open it :
image

Scenarios are serialized as XML files in the Scenario folder :
image

Once the file is shared between server and clients, we can play the scenario :
image

**Warg**
- Add footsteps sounds to wargs.
- Improved animations for wargs.
- New and improved animations for rider.

**Minor fixes/improvements**
- Fix crash caused by CapturableZone synching.
- Fix crash caused by IA controlling siege weapons when no enemy siege engines exist.
- Fix crash caused by missing flag in Scenario if none fitted the culture. We now grant a default flag.
- Better banner visuals in team selection.
- New minor factions : autochtone and explorators. They use native assets.
- Formation system is now disabled by default in Battle game mode.

### CODE REVIEW

**Comment by Pierre-HenriLemp** [comment]
image
:c

File: Alliance.Client/Core/Console/ConsoleCommands.cs [hunk 1]
```diff
@@ -2,6 +2,7 @@
 using Alliance.Common.Extensions.AdminMenu.NetworkMessages.FromClient;
 using Alliance.Common.Extensions.Audio;
 using Alliance.Common.Extensions.Audio.NetworkMessages.FromClient;
+using Alliance.Common.Extensions.FakeArmy.NetworkMessages.FromClient;
```
Comment (Pierre-HenriLemp) [inline Alliance.Client/Core/Console/ConsoleCommands.cs:5]: Est-ce que cet import est utilisé ?

Comment (Byak0) [reply Alliance.Client/Core/Console/ConsoleCommands.cs:5]: Yep la commande SpawnFakeArmy en a besoin

**Review by spawnpbbw — COMMENTED** [review]
Gateau

**Review by spawnpbbw — APPROVED** [review]
Gateau approuvé

**Pull request merged**